helper.php

```diff
@@ -220,7 +220,7 @@ public function ensureAbsoluteId($id) {
      * @param Doku_Renderer_xhtml $R
```

Comment:
Yup, I'll update that. You mean the functions in #1239? I tried to comment every changed function signature, did I miss some?